lib/cjs/vestaAPI.js

```diff
@@ -49,6 +49,11 @@ class Vesta {
     }
     postMessage(subscriptionId, postMessage) {
         return __awaiter(this, void 0, void 0, function* () {
+            if (typeof postMessage === 'string') {
+                if (containsInvalidCharacters(postMessage)) {
+                    throw new Error('Input contains one or more invalid characters.');
```

Comment:
Throwing a more helpful error for the user than the 500 that Vestaboard's API returns - there's no indication to the user from Vestaboard _why_ it threw the 500, so this solves that.

Reply:
Agreed. Bounced off of that quite a bit when getting things set up originally.